catppuccin.tmux

```diff
@@ -31,6 +31,160 @@ setw() {
+  fi
+
+  if [[ $window_status_icon_enable == "no" ]]
+  then
+    local show_window_status="#F"
+  fi
+
+  echo $show_window_status
+}
+
+build_window_format() {
+  local number="$1"
+  local color="$2"
+  local background="$3"
+  local text="$4"
+  local fill="$5"
+
+  if [[ $window_status_enable == "yes" ]]
+  then
+    local icon="$( build_window_icon )"
+    text="$text $icon" 
+  fi
+
+  if [[ $fill == "none" ]] 
+  then
+    local show_left_separator="#[fg=$thm_gray,bg=$thm_bg,nobold,nounderscore,noitalics]$window_left_separator"
+    local show_number="#[fg=$thm_fg,bg=$thm_gray]$number"
+    local show_middle_separator="#[fg=$thm_fg,bg=$thm_gray,nobold,nounderscore,noitalics]$window_middle_separator"
+    local show_text="#[fg=$thm_fg,bg=$thm_gray]$text"
+    local show_right_separator="#[fg=$thm_gray,bg=$thm_bg]$window_right_separator"
+
+  fi
+
+  if [[ $fill == "all" ]] 
+  then
+    local show_left_separator="#[fg=$color,bg=$thm_bg,nobold,nounderscore,noitalics]$window_left_separator"
+    local show_number="#[fg=$background,bg=$color]$number"
+    local show_middle_separator="#[fg=$background,bg=$color,nobold,nounderscore,noitalics]$window_middle_separator"
+    local show_text="#[fg=$background,bg=$color]$text"
+    local show_right_separator="#[fg=$color,bg=$thm_bg]$window_right_separator"
+
+  fi
+  
+  if [[ $fill == "number" ]] 
+  then
+    local show_number="#[fg=$background,bg=$color]$number"
+    local show_middle_separator="#[fg=$color,bg=$background,nobold,nounderscore,noitalics]$window_middle_separator"
+    local show_text="#[fg=$thm_fg,bg=$background]$text"
+
+    if [[ $window_number_position == "right" ]]
+    then
+      local show_left_separator="#[fg=$background,bg=$thm_bg,nobold,nounderscore,noitalics]$window_left_separator"
+      local show_right_separator="#[fg=$color,bg=$thm_bg]$window_right_separator"
+    fi
+
+    if [[ $window_number_position == "left" ]]
+    then
+      local show_right_separator="#[fg=$background,bg=$thm_bg,nobold,nounderscore,noitalics]$window_right_separator"
+      local show_left_separator="#[fg=$color,bg=$thm_bg]$window_left_separator"
+    fi
+
+  fi
+
+  local window_format
+
+  if [[ $window_number_position == "right" ]]
+  then
+    window_format="$show_left_separator$show_text$show_middle_separator$show_number$show_right_separator"
+  fi
+
+  if [[ $window_number_position == "left" ]]
+  then
+    window_format="$show_left_separator$show_number$show_middle_separator$show_text$show_right_separator"
+  fi
+
+  echo $window_format
+}
+
+build_status_module() {
+  local index="$1"
+  local icon="$2"
+  local color="$3"
+  local text="$4"
+
+  if [[ $index -eq 0 || $status_connect_separator == "no" ]]
```

Comment:
With single brackets, the and and or operators have to go outside the brackets, like so: `[ .. ] && [ .. ]`